cluster-http/src/main/resources/reference.conf

```diff
@@ -5,6 +5,13 @@
 # This is the reference config file that contains all the default settings.
 # Make your edits/overrides in your application.conf.
 
-akka.management.cluster.http {
+akka.cluster.http.management {
 
-}
+  # The hostname where the HTTP Server for Http Cluster Management will be started.
+  # This defines the interface to use.
+  hostname = "127.0.0.1"
+
+  # The port where the HTTP Server for Http Cluster Management will be bound.
+  # The value will need to be from 0 to 65535.
+  port = 19999
+}
```

Comment:
add newline at end of conf files
(important for reference.conf, because of file merging by tools like sbt-assembly)